packages/flutter_tools/test/channel_test.dart

```diff
@@ -52,6 +52,24 @@ void main() {
       expect(testLogger.statusText, contains('Flutter channels:'));
     }
 
+    Future<void> simpleChannelTestOrderedByStability(List<String> args) async {
+      final ChannelCommand command = ChannelCommand();
+      final CommandRunner<void> runner = createTestCommandRunner(command);
+      await runner.run(args);
+      expect(testLogger.errorText, hasLength(0));
+      // The bots may return an empty list of channels (network hiccup?)
+      // and when run locally the list of branches might be different
+      // so we check for the header text rather than any specific channel name.
+      final List<String> tmpChannels = testLogger.statusText.toString().split('\n');
+      final List<String> channels = tmpChannels.sublist(1, tmpChannels.length - 1);
+
+      expect(testLogger.statusText, contains('Flutter channels:'));
+      expect(channels[0], '  master');
```

Comment:
You can do `expect(something, contains('master'))` so that you don't need to worry about whitespace.